src/rivescript.coffee

```diff
@@ -940,6 +940,19 @@ class RiveScript
     return undefined
 
   ##
+  # object lastTrigger (string user)
```

Comment:
This name in the documentation should be plural to match the function name below.